coderedcms/project_template/pro/website/migrations/0003_footer_site_navbar_site.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 5.1.6 on 2025-02-17 15:16
```

Comment:
We have a special process for generating migrations in the project_template.
1. Install the oldest supported version of Django (currently 4.2).
2. Make a fresh testproject with `coderedcms start --template pro testproject`.
3. In the new testproject, delete all the migrations in `website/migrations/`
3. Run `python manage.py makemigrations`
4. Copy `website/migrations/0001_initial.py` back into `project_template/pro/website/migrations/`

This process ensures that new projects get a "clean" starting point.